ui/desktop/electron-app/src/cli/path.js

```diff
@@ -20,6 +21,16 @@ module.exports = {
       : path.resolve(process.resourcesPath, 'cli', binaryName);
     const isBuiltInCli = existsSync(builtInCliPath);
 
+    // Filter out the current working directory from the list of binaries as spawn method picks up incorrect binary path
+    if (isWindows()) {
+      const binaries = which.sync(binaryName, { nothrow: true, all: true });
+
+      const filteredBinaryPath = binaries.filter(
+        (binary) => !binary.startsWith(process.cwd()),
```

Comment:
I just noticed there's actually 3 results returned by `which`, do you also see a `boundary.exe` returned? Is that something we need to filter out as well?

Reply:
Hmm, I tested this a few times and only saw two results from `which`, any idea where we get this `boundary.exe` from?

Reply:
Hmm maybe it's just my env? I'm not exactly sure since it seemed like a relative path which I assumed was the DC itself. Either way it was the last one in the list so it wasn't getting picked so I'm not sure if it's an issue?

Would be good if someone else could confirm